src/explorer/CosmosDBTreeItem.ts

```diff
@@ -145,97 +168,185 @@ export class CosmosDBTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
         throw new Error('You must have the "Azure Databases" extension installed to perform this operation.');
     }
 
-    private detectMongoConnections(appSettings: { [propertyName: string]: string }): IDetectedConnection[] {
-        const result: IDetectedConnection[] = [];
+    private detectConnections(appSettings: { [propertyName: string]: string }): IDetectedConnection[] {
```

Comment:
> I think it might be worth putting this into its own function. Any type of DB we support from here on out is going to just have to keep getting concatenated and it's getting kind of ugly as it is.

I don't think this is what @nturinski had in mind? Before, it was a single line (even if it was a bit ugly), but now it's a 20-line method. I would prefer if we reverted it back to what it was, unless of course @nturinski can describe a better alternative that addresses his concern. For example, @nturinski - did you just want her to move this single line into it's own method (without refactoring)?:
```
const connections: IDetectedConnection[] = this.detectMongoConnections(appSettings).concat(this.detectDocDBConnections(appSettings)).concat(this.detectPostgresConnections(appSettings));
```

Reply:
Yeah, I was thinking that it could just be its own function called getAllDetectedConnections or something to that effect.  I just think it's kind of ugly and confusing as to what it's doing, so a descriptive name could have been helpful.